external_deps/build.sh

```diff
@@ -1111,22 +1175,31 @@ all_linux_arm64_default_packages='zlib gmp nettle curl sdl2 glew png jpeg webp f
 base_linux_armhf_default_packages="${base_linux_arm64_default_packages}"
 all_linux_armhf_default_packages="${all_linux_arm64_default_packages}"
 
-# Usage
-if [ "${#}" -lt "2" ]; then
-	sed -e 's/\\t/\t/g' <<-EOF
-	usage: $(basename "${BASH_SOURCE[0]}") <platform> <package[s]...>
+linux_build_platforms='linux-amd64-default linux-arm64-default linux-armhf-default linux-i686-default windows-amd64-mingw windows-amd64-msvc windows-i686-mingw windows-i686-msvc'
+macos_build_platforms='macos-amd64-default'
+all_platforms="$(echo ${linux_build_platforms} ${macos_build_platforms} | tr ' ' '\n' | sort -u)"
+
+errorHelp() {
+	sed -e 's/\\t/'$'\t''/g' <<-EOF
+	usage: $(basename "${BASH_SOURCE[0]}") [OPTION] <PLATFORM> <PACKAGE[S]...>
 
 	Script to build dependencies for platforms which do not provide them
 
+	Options:
+	\t--download-only — only download source packages, do not build them
+	\t--prefer-ours — attempt to download from unvanquished.net first
+
 	Platforms:
-	\twindows-i686-msvc windows-amd64-msvc windows-i686-mingw windows-amd64-mingw macos-amd64-default linux-amd64-default linux-i686-default linux-arm64-default linux-armhf-default
+	\t${all_platforms}
 
 	Packages:
 	\tpkgconfig nasm zlib gmp nettle curl sdl2 glew png jpeg webp freetype openal ogg vorbis opus opusfile lua naclsdk naclports wasisdk wasmtime
 
 	Virtual packages:
 	\tbase — build packages for pre-built binaries to be downloaded when building the game
 	\tall — build all supported packages that can possibly be involved in building the game
+	\tbuild-linux — build all packages buildable on linux: ${linux_build_platforms}
```

Comment:
This is listed as a package, but it's supposed to be a platform?

Reply:
Ah yes.

Reply:
Fixed.